README.md

```diff
@@ -136,30 +135,72 @@ $ yaas cluster create cluster3
 
 ```
 $ yaas cluster list
-cluster1
-cluster2
-cluster3
+cluster1 - HDP-2.3.0.0
+cluster2 - HDP-2.3.2.0
+cluster3 - HDP-2.2.6.0
+$ yaas cluster list --fields=alerts
+TODO
 ```
 
+
+###### Options
+
+* --fields
+
 ##### show
 
 ```
-$ yaas cluster show cluster1 --format=blueprint > blueprint1.json
-$ yaas cluster show cluster2
-...
+$ yaas cluster show cluster1
+Alerts
+  PASSIVE - 0
+  CRITICAL - 2
+  OK - 3
+  WARNING - 0
+Services
+  GANGLIA
+  HBASE
+  HDFS
+  HIVE
+  KNOX
+  MAPREDUCE2
+  NAGIOS
+  OOZIE
+  PIG
+  YARN
+  ZOOKEEPER
+Hosts
+  host1.com
+  host2.com
+  host3.com
+  host4.com
+  host5.com
 ```
 
-###### Options
+##### blueprint
```

Comment:
I'm not a fan of `cluster blueprint` because it means we are using 2 subcommands to cover one endpoint which is not scalable. Suppose that, in the future, all the info in a blueprint could be exported into a "redprint"... we suddenly have to implement `cluster redprint` whereas using `--format=redprint` (where format is just passed through) would just work without any modification.